examples/example.c

```diff
@@ -208,6 +211,20 @@ main(int argc, char **argv)
         sentry_capture_event(event);
     }
 
+    if (has_arg(argc, argv, "capture-transaction")) {
+        sentry_value_t tx_ctx
+            = sentry_value_new_transaction_context("I'm a little teapot");
+        sentry_transaction_context_set_operation(
+            tx_ctx, "Short and stout here is my handle and here is my spout");
```

Comment:
More of an api question, but i wonder if it makes sense to instead to always add this as a second argument to `sentry_value_new_transaction_context("op", "name")`, this would be similar to python's `start_transaction(op, name)`. it was my understanding that you really want to have `op` filled in, if we do not want to make them obligatory we can still make them accept `NULL` for them.

also happy to discuss this on the notion page if you prefer

Reply:
i don't have any particularly strong feelings on the matter - if it's more convenient to require an `op` and it lines up with the sort of behaviour we would like to encourage from users, then this can definitely be done. the change will most likely be made in https://github.com/getsentry/sentry-native/pull/619. 

i would like to call out the description on this PR, which is still applicable to the contents of its changes: the main intent of this PR is to simply expose the work i'm doing to the rest of the team, but none of it is really "PR-ready" from my POV. i can address the comments made on this diff and any discussion on parts of this that aren't already opened up in existing PRs are welcome, but given the genuine WIP nature of the changes on here some things are expected to be fairly rough (e.g. the documentation).

Reply:
probably on me for making it possible to even see those parts of the diffs though...

Reply:
i may be commenting on the wrong PRs.  apologies, i've not been paying enough attention on what the various PRs are for